### ISSUE TITLE
Failed to install add-on SunGather

### ISSUE THREAD

**Issue by Korsberg** [bohdan-s/SunGather#160]
When try ti install I got this error , any way to fix this ?

Failed to install add-on
The command '/bin/ash -o pipefail -c pip install --no-cache-dir --upgrade pycryptodomex==3.11.0 -r requirements.txt' returned a non-zero code: 1

**Comment by hossamnagy** [comment]
I ve got same error, restarting home assistant didn’t help

**Comment by davidsddh** [comment]
The command '/bin/ash -o pipefail -c pip install --no-cache-dir --upgrade pycryptodomex==3.11.0 -r requirements.txt' returned a non-zero code: 1

Is this a simple typo /bin/[b]ash?

**Comment by bohdan-s** [comment]
No. The pycryptodomex package is only source. So the docker builds on build. But HA has disabled builds. I have a fix but it requires a bit of rework and I haven’t had time to publish it yet.

**Comment by djp3k05** [comment]
we hope that you will get the time to release the fix. thank you.

**Comment by Wildvirus** [comment]
At console i got an error: 
   running build_ext
      building 'Cryptodome.Hash._MD2' extension
      creating build/temp.linux-x86_64-cpython-311
      creating build/temp.linux-x86_64-cpython-311/src
      gcc -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DPYCRYPTO_LITTLE_ENDIAN -DSYS_BITS=64 -DLTC_NO_ASM -Isrc/ -I/usr/include/python3.11 -c src/MD2.c -o build/temp.linux-x86_64-cpython-311/src/MD2.o
      error: command 'gcc' failed: No such file or directory

Perhaps it helps..... hoping to a solution
